tsa/client.go

```diff
@@ -108,7 +108,9 @@ func (client *Client) Register(ctx context.Context, opts RegisterOptions) error
 
 	defer sshClient.Close()
 
-	go client.keepAlive(ctx, sshClient, tcpConn)
+	keepAliveInterval := time.Second * 5
+	keepAliveTimeout := time.Second * 5
```

Comment:
I think we might want a more generous timeout here; sometimes workers are deployed into not-very-accomodating networks, i.e. behind the great firewall of China. 5 seconds is a bit tight.

Since we're only really trying to have a timeout that's not infinite, and the scenario we're covering is somewhat rare, could we set it to something like 5 minutes?

Reply:
Good point. Updated the timeout to 5m.